connectors/src/connectors/notion/temporal/upserts.ts

```diff
@@ -0,0 +1,243 @@
+import type { ModelId } from "@dust-tt/types";
+import {
+  executeChild,
+  ParentClosePolicy,
+  proxyActivities,
+  workflowInfo,
+} from "@temporalio/workflow";
+import type PQueue from "p-queue";
+
+import type * as activities from "@connectors/connectors/notion/temporal/activities";
+import {
+  syncResultPageChildWorkflow,
+  syncResultPageDatabaseChildWorkflow,
+} from "@connectors/connectors/notion/temporal/child_workflows";
+import { MAX_PAGE_IDS_PER_CHILD_WORKFLOW } from "@connectors/connectors/notion/temporal/config";
+
+const {
+  garbageCollectorMarkAsSeenAndReturnNewEntities,
+  fetchDatabaseChildPages,
+  upsertDatabaseStructuredDataFromCache,
+} = proxyActivities<typeof activities>({
+  startToCloseTimeout: "10 minute",
+});
+
+export async function upsertDatabase({
```

Comment:
This doesn't need to be exported right ?

Reply:
It's called from 2 files (child & admin workflows :()